apps/template/src/components/AppContent.tsx

```diff
@@ -8,11 +8,11 @@ import { Breadcrumb, BreadcrumbItem, Container } from "@cloudoperators/juno-ui-c
 
 import WelcomeView from "./WelcomeView"
 
-const AppContent = () => {
+const AppContent = (): JSX.Element => {
   return (
     <>
       <Breadcrumb>
-        <BreadcrumbItem icon="home" label="Template App Home" />
+        <BreadcrumbItem onClick={() => {}} icon={null} label="Template App Home" />
```

Comment:
**(Curious Question ?)**

Why did we change `icon` to `null` here? 
- The default icon seems to be `help`, so I think this would change styling? Wdyt?
- `icon` is already `null` in the component, by default. I don't think we need to explicitly set it here? If TS is complaining, then this is an interesting case where a default set doesn't fix the issue. 😳

Reply:
What about this?